connect/transforms/src/main/java/org/apache/kafka/connect/transforms/Cast.java

```diff
@@ -137,6 +140,7 @@ private R applySchemaless(R record) {
             String field = fieldSpec.getKey();
             updatedValue.put(field, castValueToType(null, value.get(field), fieldSpec.getValue()));
         }
+        log.trace("Schemaless SMT application with updatedValue: {}", updatedValue);
```

Comment:
I'm not sure these add value since TransformationChain has log messages with the records, which should be sufficient to know whether the schemas are null.